### DISCUSSION TITLE
Regarding the doubt about the version of the RTPS protocol

### DISCUSSION THREAD

**Discussion by yuan-ge** [OpenDDS/OpenDDS#4682] [Q&A] [answered]
This [document](https://opendds.readthedocs.io/en/latest-release/devguide/run_time_configuration.html#cfg-prop-rtps_discovery-ResendPeriod) says please refer to the document: [RTPS v2.3 8.5.3 The Simple Participant Discovery Protocol](https://www.omg.org/spec/DDSI-RTPS/2.3/PDF#page=115&view=FitH,278.68). And I went to the website and found the latest version of RTPS is 2.5. So, has the RTPS protocol currently used by OpenDDS been updated to 2.5?

**Comment by jrw972** [comment, marked as answer]
OpenDDS is currently at 2.4.

**Reply by mitza-oci** [reply]
This is correct but maybe not the complete answer.
The document published by OMG which is listed as the 2.5 version of the spec defines the most recent version of ProtocolVersion_t to be 2.4 (see table 8.2).
https://www.omg.org/spec/DDSI-RTPS/2.5
